backend/backend.go

```diff
@@ -549,16 +549,17 @@ func (backend *Backend) DeregisterKeystore() {
 	fingerprint, _ := backend.keystore.RootFingerprint()
 	backend.log.WithField("rootFingerprint", fingerprint).Info("deregistering keystore")
 	backend.keystore = nil
-	backend.Notify(observable.Event{
-		Subject: "keystores",
-		Action:  action.Reload,
-	})
 
 	backend.uninitAccounts()
 	// TODO: classify accounts by keystore, remove only the ones belonging to the deregistered
 	// keystore. For now we just remove all, then re-add the rest.
 	backend.initPersistedAccounts()
 	backend.emitAccountsStatusChanged()
+
+	backend.Notify(observable.Event{
+		Subject: "keystores",
+		Action:  action.Reload,
+	})
```

Comment:
actually this doesn't fix it, going to account->summary->unplug still shows unknown account code error

it seems that it happens only if there are many accounts (even disabled ones).

but if i wait for a long time the error doesn't happen account->summary->wait->unplug